drizzlepac/haputils/generate_custom_svm_param_file.py

```diff
@@ -0,0 +1,242 @@
+#!/usr/bin/env python
+
+import argparse
+import datetime
+import json
+import logging
+import os
+import sys
+import traceback
+
+from drizzlepac.haputils import config_utils
+from drizzlepac.haputils import ci_table
+from drizzlepac.haputils import poller_utils
+
+from stsci.tools import logutil
+
+__taskname__ = 'generate_custom_svm_param_file'
+MSG_DATEFMT = '%Y%j%H%M%S'
+SPLUNK_MSG_FORMAT = '%(asctime)s %(levelname)s src=%(name)s- %(message)s'
+log = logutil.create_logger(__name__, level=logutil.logging.NOTSET, stream=sys.stdout,
+                            format=SPLUNK_MSG_FORMAT, datefmt=MSG_DATEFMT)
+__version__ = 0.1
+__version_date__ = '04-Dec-2020'
+# ----------------------------------------------------------------------------------------------------------------------
+
+
+def make_svm_input_file(input_filename, diagnostic_mode=False, input_custom_pars_file=None,
+                        output_custom_pars_file=None, phot_mode="both",
+                        clobber=False, log_level=logutil.logging.INFO):
+    """
+    Run the HST Advanced Products (HAP) generation code.  This routine is the sequencer or
+    controller which invokes the high-level functionality to process the single visit data.
+
+    Parameters
+    ----------
+    input_filename: string
+        The 'poller file' where each line contains information regarding an exposures taken
+        during a single visit.
+
+    diagnostic_mode : bool, optional
+        Allows printing of additional diagnostic information to the log.  Also, can turn on
+        creation and use of pickled information.
+
+    input_custom_pars_file: string, optional
```

Comment:
The presence of this parameter is a little confusing since this function is designed to create this parameter (essentially).

Reply:
Issue resolved by commit [e39100a](https://github.com/spacetelescope/drizzlepac/pull/896/commits/e39100a3e033d8707953af071843d230a98941f9)